lib/container.js

```diff
@@ -8,6 +8,9 @@ const recorder = require('./recorder');
 const event = require('./event');
 const WorkerStorage = require('./workerStorage');
 const store = require('./store');
+const { getTestRoot } = require('../lib/command/utils');
```

Comment:
I'm against including this into a container
container should not know anything about filesystem

Reply:
Just solved it to myself by putting of `global.codecept_dir = testsPath;` after https://github.com/codeceptjs/CodeceptJS/blob/3.x/lib/command/generate.js#L26.

But still there is an issue that all custom helpers need to have included `const Helper = require('@codeceptjs/helper');`, otherwise you get the issue https://github.com/codeceptjs/CodeceptJS/issues/3512#issuecomment-1335554742

Reply:
> But still there is an issue that all custom helpers need to have included const Helper = require('@codeceptjs/helper');, otherwise you get the issue https://github.com/codeceptjs/CodeceptJS/issues/3512#issuecomment-1335554742

On the other hand

- Newly created Helpers (by `codeceptjs gh`) have the variable `Helper` set
- I can fix my existing custom helpers easily
- As for other custom helpers that I use, only `codeceptjs-chai` doesn't have `Helper` set and I must patch it. Others (`codeceptjs-cmdhelper`, `codeceptjs-dbhelper` and `codeceptjs-pixelmatchhelper`) are already set correctly.

So it's not a big issue.

Reply:
@mirao, how did you patch `codeceptjs-chai` helper?

Reply:
@zhigulin-n 

`codeceptjs-chai@2.3.2.patch`:
```patch
diff --git a/index.js b/index.js
index bd248fce8c1581fe07a63abae3a469159e7891ca..7cd6dcca0e8ba1e47195e8053f07187ca08782ca 100644
--- a/index.js
+++ b/index.js
@@ -1,3 +1,5 @@
+const Helper = require('@codeceptjs/helper');
+
 /* eslint-disable global-require */
 const chai = require( 'chai' );
 
```

Reply:
@kobenguyent Thanks. Your PR really helps to solve the issue and therefore my patch is not needed anymore.
I tested that then `codeceptjs run` and `codeceptjs gt` work well in a project using `codeceptjs-chai`.